### PR TITLE
Add tidl Rust description

### PR DESCRIPTION
**Pull request by dk3776**
### Change Description ###

 - Add tidl Rust description

### CODE REVIEW

File: docs/application/native/guides/app-management/tidl.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Usage:
 > [!NOTE]
 > Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.
 > Generating cion option with `-c` or `--cion` is supported since Tizen 6.5.
+> Generating Rust language type is only supported in version 2.
```
Comment (jusung-son) [inline docs/application/native/guides/app-management/tidl.md:32]: Please add 'Rust' keyword
-l, --language=LANGUAGE Select generating language (C, C++, C#, Java(CION only)).

Comment (jusung-son) [reply docs/application/native/guides/app-management/tidl.md:32]: version 2. ==> protocol version 2.

Comment (dk3776) [reply docs/application/native/guides/app-management/tidl.md]: done

File: docs/application/native/guides/app-management/tidl.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Usage:
 > [!NOTE]
 > Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.
 > Generating cion option with `-c` or `--cion` is supported since Tizen 6.5.
+> Generating Rust language type is only supported in protocol version 2.
```
Comment (upple) [inline docs/application/native/guides/app-management/tidl.md:32]: It should be specified that it is supported since Tizen 9.0.

Comment (kdk3776) [reply docs/application/native/guides/app-management/tidl.md]: done

Comment (dk3776) [reply docs/application/native/guides/app-management/tidl.md]: done

**Review by jusung-son — APPROVED** [review]

**Review by upple — APPROVED** [review]

**Review by safir-srbd — APPROVED** [review]
LGTM!

**Pull request merged**